packages/web-console/src/scenes/Notifications/Notification/SuccessNotification/index.tsx

```diff
@@ -41,22 +48,61 @@ const ZapIcon = styled(Zap)`
   color: ${color("draculaYellow")};
 `
 
+const CopyIcon = styled(Copy)`
+flex-shrink: 0;
+`
+
 export const SuccessNotification = (props: NotificationShape) => {
-  const { createdAt, content, sideContent, jitCompiled } = props
+  const { createdAt, content, sideContent, jitCompiled, request, result } = props
+  const handleCopyToClipboard = useCallback(() => {
+    let clipboardText = `[${format("HH:mm:ss", createdAt)}] ${request?.query}\n`
+    if (result?.timings) {
+      try {
+        const { count, compiler, fetch, execute } = result.timings;
+        clipboardText += `✔ Success: ${result.count?.toLocaleString()} rows in ${formatTiming(fetch)}\n`;
```

Comment:
to be honest, i'm not really sure if copy to clipboard should format as a table. Perhaps it's enough to copy same text that is visible in UI (except not truncate it). WDYT?

nonetheless, if we'd continue with formatting as a table, then i'd suggest to not use a dependency but build a util.

this should do the trick:

```
const row1 = ["Execute", "Network", "Total", "Count", "Compile"];
const row2 = ["6.07ms", "86.93ms", "93ms", "0", "515.97µs"];
const rows = [row1, row2];

const renderTable = (rows) => {
  const colMax = rows[0].map((_, i) =>
    Math.max(...rows.map((row) => String(row[i]).length))
  );

  const header = `| ${rows[0]
    .map((col, i) => col.padEnd(colMax[i]))
    .join(" | ")} |`;

  const separator = `+${colMax.map((max) => "-".repeat(max + 2)).join("+")}+`;

  const data = rows
    .slice(1)
    .map(
      (row) =>
        `| ${row.map((col, i) => String(col).padEnd(colMax[i])).join(" | ")} |`
    );

  return [separator, header, separator, ...data, separator].join("\n");
};

console.log(renderTable(rows));

/*

+---------+---------+-------+-------+----------+
| Execute | Network | Total | Count | Compile  |
+---------+---------+-------+-------+----------+
| 6.07ms  | 86.93ms | 93ms  | 0     | 515.97µs |
+---------+---------+-------+-------+----------+

*/

```

cc @insmac wdyt?

Reply:
I'm not sure what the most common use case would be for copying the data out. I have updated it to use this code rather than the package. If it is for debugging it may be nice to pre wrap it in `code` backticks for Slack or the like.

Reply:
@argshook 

I think `Copy to clipboard` should format in a way that's easy to read when pasted in channels like Slack or GitHub (when creating a new issue, for example).

For one, it's very hard to view partial screenshots (i.e. showing the notification log) on mobile, and even sometimes on desktop - even the thumbnails are small. 
Additionally, you can copy and paste further in a more convenient manner.
This follows the pattern of QuestDB REST API showing operation status (like after importing a CSV file) as an ASCII table by default, unless JSON format is requested 🙂 

In short: I like this approach!